client/src/pages/PackagesScreen/MatchHeader/MatchHeader.tsx

```diff
@@ -0,0 +1,26 @@
+import { ArrowRightOutlined, EnvironmentOutlined } from '@ant-design/icons';
+import styles from './match-header.module.scss';
+import { Tag, Typography } from 'antd';
+import { formattedDate } from '@/utils/date.utils';
+
+const { Text } = Typography;
+
+interface MatchHeaderProps {
+    from: string;
+    to: string;
```

Comment:
```suggestion
    startDate: string;
    endDate: string;
```
Need to know it represents a date. and from and to is a bit general